client/jsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
jsconfig.json is a vscode thing that allows you to define configuration details for individual folders. I don't think we can do that with `/.vscode/settings.json`